Library/RewardStateTests.swift

```diff
@@ -168,4 +168,59 @@ class RewardStateTests: TestCase {
       RewardCellProjectBackingState.state(with: project, reward: reward)
     )
   }
+
+  func test_noRewardType_BackedNonLive() {
+    let reward = Reward.noReward
+      |> Reward.lens.remaining .~ 0
+      |> Reward.lens.limit .~ 0
+
+    let backing = Backing.template
+      |> Backing.lens.reward .~ reward
+
+    let project = .template
+      |> Project.lens.personalization.isBacking .~ true
+      |> Project.lens.personalization.backing .~ backing
+      |> Project.lens.state .~ .successful
+
+    XCTAssertEqual(
+      .backed(live: .nonlive, rewardState: .inactive),
+      RewardCellProjectBackingState.state(with: project, reward: reward)
+    )
+  }
+
+  func test_noRewardType_BackedLive() {
+    let reward = Reward.noReward
+      |> Reward.lens.remaining .~ 0
+      |> Reward.lens.limit .~ 0
+
+    let backing = Backing.template
+      |> Backing.lens.reward .~ reward
+
+    let project = .template
+      |> Project.lens.personalization.isBacking .~ true
+      |> Project.lens.personalization.backing .~ backing
+      |> Project.lens.state .~ .successful
```

Comment:
Shouldn't this be `.live` since this test is testing for `noRewardType_backedLive`?

Reply:
No, it's correct. Forgot to rename.